Cargo.toml

```diff
@@ -81,6 +81,7 @@ default = [
     "time",
     "toml",
     "url",
+    "batchnoise",
```

Comment:
abc order

Reply:
Good catch, my bad. Was following the below list, but it makes a lot of sense now that I look at it.